public/scripts/alert.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const notifications = document.querySelector('#notifications')
+
+module.exports = {
+  alert(message) {
+    notifications.textContent = message
+    notifications.style.display = 'flex'
+    setTimeout(function() {
+      notifications.style.display = 'none'
+    }, 5000)
+  }
```

Comment:
You should set this up so you can clear the timeout every time this is called. See https://github.com/ZakMiller/TheEmpire/blob/c6bec7f481318b462f43c4698b10502670f53e44/public/scripts/loginHandler.js#L44-L53 for an example.

What you have here acts funky if this function is called multiple times within a 5 second window (you can see what I mean if you spam enter on an empty sentence a few times)